app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CartItemsController < ApplicationController
+  before_action :set_cart_item, only: %i[update destroy]
+
+  def create
+    @item = Item.find(params[:item_id])
```

Comment:
`@item` にする必要はなくて、`item`で良さそう。
不必要にインスタンス変数を増やさないようにしたい

Reply:
```suggestion
    @item = Item.pulished.find(params[:item_id])
```
publishedに限定しておかないと、非公開の商品をカートに入れれちゃうのでは

Reply:
という対応をした後に、
cartに追加しようとした直前に非公開にされたらfindで例外になる
(レアケースだからあえて無視しているのか)